code/modules/shuttle/marine_dropship.dm

```diff
@@ -261,8 +239,11 @@
 
 /obj/docking_port/mobile/marine_dropship/one
 	name = "Alamo"
-	id = SHUTTLE_ALAMO
+	id = SHUTTLE_DROPSHIP
 	control_flags = SHUTTLE_MARINE_PRIMARY_DROPSHIP
+	callTime = 30 SECONDS
+	rechargeTime = 5 SECONDS
+	prearrivalTime = 10 SECONDS
```

Comment:
Imo this recharge time should be way higher, my understanding is this is the more static one and the tadpole is the leapfrogging one

Reply:
On the contrary, I wanted the Alamo to be the leap frogger! It's a dropship, it should be delivering troops or picking them up and zooming away in my opinion. If the recharge is too long you will have people trying to use it as a mobile FOB like they did with the Tadpole. Just land and never move again... which defeats the purpose of a dropship!